packages/mui-base/src/useSlider/useSlider.ts

```diff
@@ -296,6 +297,89 @@ export function useSlider(parameters: UseSliderParameters): UseSliderReturnValue
       otherHandlers?.onBlur?.(event);
     };
 
+  const changeValue = (event: React.KeyboardEvent | React.ChangeEvent, valueInput: number) => {
```

Comment:
I've extracted the previous `createHandleHiddenInputChange` logic in a new method that can be re-used in two places now. The only difference is how we read the value (from the input target, or calculating it in case the PageUp/PageDown arrows are used.